.gitignore

```diff
@@ -1,5 +1,11 @@
 # file types to ignore
-
+.*
+!/.gitignore
+!/.travis/
+!/.travis.yml
```

Comment:
is travis even used anymore?

Reply:
there is the .travis folder, Seeing its contents, for me it can be removed

Reply:
Let's keep it for now.

I will make a separate PR to remove .travis and see what he developers think about it.

Reply:
Here it is: #7238